packages/react-components/src/images/events-placeholder.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
SVG belongs in `icons` (raw files in `images`)

Reply:
(other one as well)

Reply:
It was my first approach, but it's not an icon 😛.

Reply:
A distinction based on visual content is pretty arbitrary though, what the `Icon`/`Image` suffix is trying to achieve is that you know what you've got when you `import { *Image, *Icon } from '@asap-hub/react-components'`

Reply:
Was just messing with you 🙂